ray-operator/config/openshift/ray_operator_scc.yaml

```diff
@@ -0,0 +1,11 @@
+kind: SecurityContextConstraints
+apiVersion: security.openshift.io/v1
+metadata:
+  name: run-as-ray-user
+seLinuxContext:
+  type: MustRunAs
+runAsUser:
+  type: MustRunAs
+  uid: 1000
+users:
+  - 'system:serviceaccount:$(namespace):kuberay-operator'
```

Comment:
Where is the namespace variable set?

Reply:
Good spot, thanks @astefanutti 
The namespace is set within the default kustomize file, however, this won't translate to the openshift kustomize.